src/libstd/collections/hash/map.rs

```diff
@@ -691,6 +691,16 @@ impl<K, V, S> HashMap<K, V, S>
     /// Returns a reference to the map's [`BuildHasher`].
     ///
     /// [`BuildHasher`]: ../../std/hash/trait.BuildHasher.html
+    ///
+    /// # Examples
```

Comment:
I think you might need a newline after this for rustdoc+markdown to treat it as a doc test